jsonnet/rules.libsonnet

```diff
@@ -360,8 +360,9 @@ function(params) {
           // Should be used to suppress alerts during control plane upgrades or disruption.
         },
         {
-          expr: 'max(max_over_time(cluster_monitoring_operator_last_reconciliation_successful[1h])) == 0',
+          expr: 'max_over_time(cluster_monitoring_operator_last_reconciliation_successful[5m]) == 0',
```

Comment:
We have a resync period of 15m - https://github.com/openshift/cluster-monitoring-operator/blob/master/pkg/operator/operator.go#L111. This means the alert may be flapping as we are taking shorter range than resync interval.

Reply:
NVM, I was mistaken how this works

/lgtm